src/helpers/web3.ts

```diff
@@ -0,0 +1,10 @@
+import assert from 'assert'
+import Web3 from 'web3'
+
+import Logger from 'helpers/Logger'
+const log = new Logger('helpers:web3')
+const nodeUrl = process.env.NODE_URL
+
+log.info('Connecting to ethereum using web3: %s', nodeUrl)
+assert(nodeUrl, 'NODE_URL env var is required')
+export const web3: Web3 = new Web3(nodeUrl as string)
```

Comment:
`assert` should be catching the right type... is it the TS `assert` from `dex-js` or `node`?

Reply:
It's from node. I think it's good if we use yours. I'll open an issue to address it in further PRs.

Reply:
https://github.com/gnosis/dex-js/issues/29